dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerAPIImpl.java

```diff
@@ -430,6 +441,12 @@ public Container save(Container container, List<ContainerStructure> containerStr
 			}
 		}
 
+		try {
+			String bla = null;
+		} catch (Exception e) {
+			e.getMessage();

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 13. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AIndentationCheck)